analyze.go

```diff
@@ -1595,6 +1595,7 @@ var directives = map[string][]uint{
 	"set_real_ip_from": {
 		ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxConfTake1,
 		ngxStreamMainConf | ngxStreamSrvConf | ngxConfTake1,
+		ngxMailMainConf | ngxMailSrvConf | ngxConfTake1,
```

Comment:
is this something that is supported? have you tried this out?

https://nginx.org/en/docs/stream/ngx_stream_realip_module.html based on this it is just for stream and server but does not mention `server` under `mail`?

Reply:
https://nginx.org/en/docs/mail/ngx_mail_realip_module.html  and https://github.com/nginx/nginx/blob/ef96f5835468ff8d40df29b0ddbc04ec1e5e1582/src/mail/ngx_mail_realip_module.c#L30 says it `"set_real_ip_from":`also for mail and server

Reply:
aah. I see it. it's the mail module. Thanks for sharing this @xynicole